stdlib/Sockets/src/addrinfo.jl

```diff
@@ -241,21 +256,23 @@ function getipaddr()
             rv = IPv4(ntoh(ccall(:jl_sockaddr_host4, UInt32, (Ptr{Cvoid},), sockaddr)))
             ccall(:uv_free_interface_addresses, Cvoid, (Ptr{UInt8}, Int32), addr, count)
             return rv
-        # Uncomment to enbable IPv6
-        #elseif ccall(:jl_sockaddr_in_is_ip6, Int32, (Ptr{Cvoid},), sockaddr) == 1
-        #   host = Vector{UInt128}(undef, 1)
-        #   ccall(:jl_sockaddr_host6, UInt32, (Ptr{Cvoid}, Ptr{UInt128}), sockaddrr, host)
-        #   return IPv6(ntoh(host[1]))
         end
     end
     ccall(:uv_free_interface_addresses, Cvoid, (Ptr{UInt8}, Int32), addr, count)
     return lo_present ? localhost : error("No networking interface available")
+    =#
```

Comment:
This commented out code should just be deleted rather than commented out.

Reply:
Agreed, was just keeping it around so that I don't have to fish it out of git again if I found I needed it :smile: